blog/authors.yml

```diff
@@ -3,3 +3,9 @@ gagdiez:
   title: Docs Maintainer
   url: https://github.com/gagdiez
   image_url: https://github.com/gagdiez.png
+
+saketh:
```

Comment:
I'd actually prefer not to be named as a primary author; you've done 99% of the work here!